### PR TITLE
docs: Improving readme

### PR DESCRIPTION
**Pull request by perlajarillo**
In this PR we are including in the Readme the installation process, development notes, contributors and  contributors guidelines.

Related to #44

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,85 @@
...
 
-### `npm run build`
+The file package.json contains the complete list of dependencies for this project.
 
-Builds the app for production to the `build` folder.<br>
-It correctly bundles React in production mode and optimizes the build for the best performance.
+### 2. Create a project in Firebase
+
+In order to try this app you may create a new project in the Firebase Console. You can create a free account (here) [https://console.firebase.google.com/]
+
```
Comment (Fa-v) [inline README.md:22]: The parenthesis and the square brackets are mixed up it has to be like this: `You can create a free account [here] (https://console.firebase.google.com/)` 😄

File: README.md [hunk 2]
```diff
@@ -1,41 +1,85 @@
...
+Next you will need to create a folder to storage the pictures of the mentors participating in FLAD-Mentorship program.
+
+- Go to the [Firebase console](https://console.firebase.google.com/), open the Storage section.
+- On the Files tab, click in the icon "create folder".
+- Name your new folder **images**
+
+### 5. Set the environment variables
+
+Once you have created your project in Firebase, create manually the file .env.production in the root of your project. Type the keys generated to connect to your database as follow:
```
Comment (Fa-v) [inline README.md:43]: Shall we highlight the files's name? `.env.production`
Do we need the `.env.dev` or just the production one?

**Review by Fa-v — CHANGES_REQUESTED** [review]
I think this makes a great README.
Since we're are opening the possibility for contributions, do you think is a good idea to specify an OS license?

File: README.md (hunk 1 above)
Comment (perlajarillo) [reply README.md:22]: Oh 😄! this one was trying to escape! Fixed it! 👍

File: README.md (hunk 2 above)
Comment (perlajarillo) [reply README.md:43]: I think collaborators will need just .env.production, but as a good practice, we can recommend both. I made the changes. 😉

**Comment by perlajarillo** [comment]
> I think this makes a great README.
> Since we're are opening the possibility for contributions, do you think is a good idea to specify an OS license?

I agree with you. [MIT](https://opensource.org/licenses/MIT) would be a good fit for us, what do you think?

**Comment by Fa-v** [comment]
The MIT license is perfect! :) Sorry for the delay in my response.

**Comment by perlajarillo** [comment]
Thanks Fabiola, don't worry about it. Let's merge this PR! 😄

**Pull request merged**